### PR TITLE
pass payload as json string

### PR DESCRIPTION
**Pull request by kylecjo**

### CODE REVIEW

File: src/thermal_tracker.py [hunk 1]
```diff
@@ -56,12 +56,12 @@ def send_lora(delay):
...
                 child.terminate()
         loraproc = Process(
-            target=transmit, name='lora_proc', args=(payload, ))
+            target=transmit, name='lora_proc', args=(json.dumps(payload), ))
         loraproc.start()
         time.sleep(delay)
 
 
-payload = ''
+payload = {'a': 0, 'o': 0, 'c': 0}
```
Comment (j-rewerts) [inline src/thermal_tracker.py:66]: What's a, o and c?

File: src/ttn/thethingsnetwork-send-v1.cpp [hunk 2]
```diff
@@ -114,13 +114,13 @@ void onEvent(ev_t ev)
 // used to read data from python program
 void readData()
 {
-    char buf[16];
+    char buf[51];
```
Comment (j-rewerts) [inline src/ttn/thethingsnetwork-send-v1.cpp:117]: Please handle the error when our data is larger than 51 characters.

Comment (j-rewerts) [reply src/ttn/thethingsnetwork-send-v1.cpp:117]: Throw an exception

**Review by j-rewerts — CHANGES_REQUESTED** [review]
Lets add error handling in Python for our C++ process. We should be able to see exceptions in C++ when we look in JournalCtl.

File: src/ttn/thethingsnetwork-send-v1.cpp (hunk 2 above)
Comment (kylecjo) [reply src/ttn/thethingsnetwork-send-v1.cpp:117]: will create new issue for exception handling

**Review by j-rewerts — APPROVED** [review]

**Pull request merged**